<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionReload"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>空间域</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>平滑滤波</string>
     </property>
     <addaction name="actionAvergeFilter"/>
     <addaction name="actionMedianFilter"/>
     <addaction name="actionGaussianFilter"/>
    </widget>
    <addaction name="actionGetPath"/>
    <addaction name="actionGrayChange"/>
    <addaction name="menu_3"/>
    <addaction name="actionSharpen"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>频率域</string>
    </property>
    <addaction name="actionFT"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>形态学处理</string>
    </property>
    <addaction name="actionCorrosion"/>
    <addaction name="actionExpansion"/>
    <addaction name="actionOpen_2"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>添加噪声</string>
    </property>
    <addaction name="actionAddGuassianNoise"/>
    <addaction name="actionSPNoise"/>
   </widget>
   <widget class="QMenu" name="menuGg">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionIDE"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menuGg"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/saveAs.png</normaloff>:/images/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为...</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/redo.png</normaloff>:/images/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重新载入图片</string>
   </property>
  </action>
  <action name="actionGetPath">
   <property name="text">
    <string>细化</string>
   </property>
  </action>
  <action name="actionGrayChange">
   <property name="text">
    <string>灰度反转</string>
   </property>
  </action>
  <action name="actionAvergeFilter">
   <property name="text">
    <string>均值滤波</string>
   </property>
  </action>
  <action name="actionMedianFilter">
   <property name="text">
    <string>中值滤波</string>
   </property>
  </action>
  <action name="actionSharpen">
   <property name="text">
    <string>锐化</string>
   </property>
  </action>
  <action name="actionFT">
   <property name="text">
    <string>傅里叶变换</string>
   </property>
  </action>
  <action name="actionCorrosion">
   <property name="text">
    <string>膨胀</string>
   </property>
  </action>
  <action name="actionExpansion">
   <property name="text">
    <string>腐蚀</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>开操作</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>闭操作</string>
   </property>
  </action>
  <action name="actionAddGuassianNoise">
   <property name="text">
    <string>高斯噪声</string>
   </property>
  </action>
  <action name="actionSPNoise">
   <property name="text">
    <string>椒盐噪声</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionIDE">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/about.png</normaloff>:/images/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于DIPTool</string>
   </property>
  </action>
  <action name="actionGaussianFilter">
   <property name="text">
    <string>高斯滤波</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
